client/src/components/code/CodeBlock.tsx

```diff
@@ -0,0 +1,63 @@
+import { makeStyles, Theme } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import CopyButton from '../advancedSearch/CopyButton';
+import SyntaxHighlighter from 'react-syntax-highlighter';
+import { docco } from 'react-syntax-highlighter/dist/esm/styles/hljs';
```

Comment:
what is docco ?

Reply:
`docco` is default highlight theme. We can change it to others.